theano/tensor/signal/pool.py

```diff
@@ -113,30 +114,103 @@ def pool_2d(input, ds, ignore_border=None, st=None, padding=(0, 0),
     return tensor.reshape(output, outshp, ndim=input.ndim)
 
 
-class Pool(OpenMPOp):
+def pool_3d(input, ds, ignore_border=None, st=None, padding=(0, 0, 0),
```

Comment:
make ignore_border defaul to True. It is the new interface we want to use. As this is a new user interface function, I think it would be better to use the new interface for that parameter.